### ISSUE TITLE
Catch all polyfill for IE

### ISSUE THREAD

**Issue by mcorkum** [ravendesignsystem/rds#71]
We're running into a few JS issues with IE11. I've done some research and my suggestion is to add this to the header: 

```
 <!--[if IE]>
 <script crossorigin="anonymous" src="https://polyfill.io/v3/polyfill.min.js"></script>
 <![endif]-->
```
 
This polyfill script detects the browser, and returns the missing polyfills needed for that browser.  The if IE statement only works for older browsers, so this should prevent edge from calling the external JS. 

This would keep us covered in the future for all JS polyfills for IE11. 

Alternatively we could detect the browser in JS: https://stackoverflow.com/questions/21825157/internet-explorer-11-detection

**Comment by mcorkum** [comment]
Also there's an NPM package https://babeljs.io/docs/en/babel-polyfill

Not sure how this interacts with the rest of the code, or if it's better to keep it IE specific. Although, something that scans your browser for missing compatibilities and adds them sounds pretty great if it's not heavy.

**Comment by mcorkum** [comment]
Need opinions here on what is best.

**Comment by dannybrown73** [comment]
I'll look to have a look asap. polyfill.io is likely all that is needed for js.

**Comment by mieradi** [comment]
@dannybrown73 what is actually needing a polyfill within the project? I can implement the standalone code for those instead of adding a whole package. I imagine it's just class additions and removals as per our last convo?

**Comment by marceloglacial** [comment]
@mieradi Maybe we can use Babel and Differential serving:
https://css-tricks.com/differential-serving/
https://babeljs.io/docs/en/babel-polyfill 

> As of Babel 7.4.0, this package has been deprecated in favor of directly including core-js/stable (to polyfill ECMAScript features) and regenerator-runtime/runtime (needed to use transpiled generator functions):

import "core-js/stable";
import "regenerator-runtime/runtime";

**Comment by mieradi** [comment]
@marceloglacial yeah we can include the whole thing, I just want to see what we need a polyfill for. If it's only one or two things, I can just include those fills and not have to run the whole dependancy. Regardless, it's not that heavy.

**Comment by dannybrown73** [comment]
I don't know. @mcorkum reported js errors in IE11 but I don't see them in the console. @mcorkum can you paste the JS errors in IE11.

**Comment by mieradi** [comment]
@troychaplin79 When you finish the Webpack setup, add this to the setup. This should fix your issue, and if not we can import it to the index.js and be done with it. 
https://babeljs.io/docs/en/babel-polyfill

**Comment by dannybrown73** [comment]
Does the deprecation notice matter for our use case?

**Comment by troychaplin** [comment]
I could be wrong, but @mcorkum after the stuff you worked on this morning, this is no longer needed, correct?

**Comment by dannybrown73** [comment]
I should also mention, there are polyfills imported already into the project to resolve previous ie issues. If there aren't any further IE JS related issues, there's a chance we do not need this.

**Comment by troychaplin** [comment]
There's only 1 final IE11 issue to fix, as noted in issue #70, it's not looking like we'll need any additional polyfills. Going to close this for now, and we'll revisit additional polyfills is necessary
